WORKSPACE

```diff
@@ -33,8 +33,9 @@ load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
 rbe_autoconfig(
     name = "rbe_default",
     exec_properties = create_exec_properties_dict(
-        docker_add_capabilities = "SYS_PTRACE",
+        docker_add_capabilities = "SYS_PTRACE,NET_ADMIN",
         docker_privileged = True,
+        docker_run_as_root = True,
```

Comment:
While setting these options might be useful for a subset of tests, I don't feel like it's a good idea to use them for the 99 % tests that really don't need them. Tests should run in as restricted environments as possible. and giving them more capabilities (and root access) doesn't sound like a good practice.

Reply:
This is a hard requirement in order for this test to work, unfortunately.

See conversation in  https://groups.google.com/a/google.com/g/rbe-users/c/hWO-3O9sHWA for background. The main thing is that in order to modify the network stack of the docker containers, the binary needs to have NET_ADMIN capability (which is impossible to have right now without running as uid 0 within the container).

If there's some way that I can set this for only the test that needs it (maybe using some kind of one-off bazel RBE suite), I'd be fine with that.